src/generator/__snapshots__/errata.integration.js.snap

```diff
@@ -241,7 +241,7 @@ exports[`aggregates multiple errata-based issues into advisory/cve plays 1`] = `
   become: true
   tasks:
     - name: check for update
-      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q --cve {{ insights_issues | regex_search('(CVE-[0-9]{4}-[0-9]+( --cve CVE-[0-9]{4}-[0-9]+)*)') }}\\"
+      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q {{ insights_issues | regex_search('(--cve (CVE-[0-9]{4}-[0-9]+))+') }}\\"
```

Comment:
@semtexzv I was looking through the Regex to make sure that we are looking for the correct strings, and it's come to my attention that we probably need to change the CVE Regex one more time.

i'm thinking that it has to look something like this --> `([ ]*(--cve (CVE-[0-9]{4}-[0-9]+)))+`

This allows for the whole string i.e. (--cve CVE-1234-1231 --cve CVE-1235-1234) is included in the playbook.